### ISSUE TITLE
Undefined class constant 'CAUSE_EXPLOSION'

### ISSUE THREAD

**Issue by nasiridrishi** [TheAz928/VanillaEnchantments#18]
```
[Server thread/CRITICAL]: Could not pass event 'pocketmine\event\entity\EntityDamageEvent' to 'VanillaEnchantments v2.0.0-dev': Undefined class constant 'CAUSE_EXPLOSION' on VanillaEnchantments\handlers\BlastProtection
[12:36:32] [Server thread/CRITICAL]: Error: "Undefined class constant 'CAUSE_EXPLOSION'" (EXCEPTION) in "VanillaEnchantments_v2.0.0-dev.phar/src/VanillaEnchantments/handlers/BlastProtection" at line 22
```
PMMP 1.7 dev473, API3 Alpha10
VanillaEnchantments v2.0.0-dev

**Comment by inxomnyaa** [comment]
Duplicate of https://github.com/TheAz928/VanillaEnchantments/issues/17

**Comment by nasiridrishi** [comment]
How much latest I use???

**Comment by inxomnyaa** [comment]
Reason: missing import. Fixed in PR #20

**Comment by nasiridrishi** [comment]
Ty
